R/extract_lhs.R

```diff
@@ -35,10 +35,14 @@ extract_lhs2 <- function(model, ...) {
 #' @return A character string
 #' @noRd
 
-extract_lhs.lm <- function(model, ital_vars, ...) {
+extract_lhs.lm <- function(model, ital_vars,
+                           show_distribution,
+                           use_coefs, ...) {
   lhs <- rownames(attr(model$terms, "factors"))[1]
 
   lhs_escaped <- escape_tex(lhs)
+  if (use_coefs){
+  lhs_escaped <- add_hat(lhs)}
   add_tex_ital_v(lhs_escaped, ital_vars)
```

Comment:
Again super minor, but can you indent these two lines two spaces?

Reply:
Fixed